src/client/unity.yml

```diff
@@ -5,6 +5,187 @@ info:
 servers:
   - url: /unity/v2 # to be determined
 paths:
+  "/billing/account":
+    get:
+      operationId: GetAccount
+      tags:
+        - Account
+      responses:
+        "200":
+          description: Account info for billing
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Account"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  "/billing/notify_settings":
+    get:
+      operationId: GetBillingNotifySettings
+      tags:
+        - BillingNotifySettings
+      responses:
+        "200":
+          description: Settings for billing notifications
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BillingNotifySettings"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    put:
+      operationId: PutBillingContact
+      tags:
+        - PaymentMethods
+      requestBody:
+        description: Billing Contact to Update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/BillingNotifySettings"
+      responses:
+        "200":
+          description: billing contact updated
```

Comment:
Looks like some copy-pasta in this `put` definition; several references to billing contact and/or PaymentMethods, but this is about notification settings.